watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/posttreatment_ideal_naocl_chlorination_block.py

```diff
@@ -98,8 +98,8 @@
 # Import the idaes object for the EquilibriumReactor unit model
 from idaes.generic_models.unit_models.equilibrium_reactor import EquilibriumReactor
 
-# Import the WaterTAP object inherited for the Mixer unit model
-from watertap.examples.flowsheets.full_treatment_train.model_components import Mixer
+# Import the Mixer unit model
+from idaes.generic_models.unit_models import Mixer
```

Comment:
I am not sure how we want to manage the shift to IDAES v2.0.0.a2 (i.e. the next release of IDAES), but we are reorgansing the repository right now and these have been moved to `idaes.models.unit_models`.

There are deprecation paths in place so this will continue to work through November, but we will need to transition to the new structure at some point.